### DISCUSSION TITLE
Cant start container anymore

### DISCUSSION THREAD

**Discussion by costaht** [FreshRSS/FreshRSS#5611] [Q&A] [answered]
Anyone else getting these errors when trying to start your container?

```
OIDC_ENABLED): -c: line 2: unexpected EOF while looking for matching `)'
OIDC_ENABLED): -c: line 2: unexpected EOF while looking for matching `)'
OIDC_ENABLED): -c: line 2: unexpected EOF while looking for matching `)'
OIDC_ENABLED): -c: line 2: unexpected EOF while looking for matching `)'
OIDC_ENABLED): -c: line 2: unexpected EOF while looking for matching `)'
OIDC_ENABLED): -c: line 2: unexpected EOF while looking for matching `)'
OIDC_ENABLED): -c: line 2: unexpected EOF while looking for matching `)'
OIDC_ENABLED): -c: line 2: unexpected EOF while looking for matching `)'
OIDC_ENABLED): -c: line 2: unexpected EOF while looking for matching `)'
OIDC_ENABLED): -c: line 2: unexpected EOF while looking for matching `)'
OIDC_ENABLED): -c: line 2: unexpected EOF while looking for matching `)'
```

**Comment by Alkarex** [comment]
Hello,
Can you tell a bit more about your platform, and which Docker image you are using?
Show also your docker-compose or docker run

**Reply by costaht** [reply]
Sure. I'm running `freshrss/freshrss:edge@sha256:b09de73b53bb6b459703bdf4bbeb9a9007080e003ba110d0649b7732aeb8fc8b` and my host is running CentOS Stream release 9.

Once I setup Freshrss I never touched the container again. I just use Watchtower to keep it up to date, and unfortunately those are the only logs I get from the container.

**Reply by Alkarex** [reply]
I cannot reproduce. Could you try to just run the container manually on your platform?

```sh
docker run --rm freshrss/freshrss:edge
```

A workaround can be to use our alternative Alpine image: `freshrss/freshrss:edge-alpine`

**Reply by costaht** [reply]
I was recreating the container with portainer and for some reason it never fixed the problem. So a manually deleted the container and recreated it and now it's working. Go figure...
Thank you!

**Reply by RCdeWit** [reply]
Running into the same error with Portainer, but only when setting the `CRON_MIN` env var. Have tried recreating the container, but running into the same issue. Any ideas?

- image: `freshrss/freshrss:latest`

```
OIDC_ENABLED): -c: line 2: unexpected EOF while looking for matching `)'
```

**Reply by Alkarex** [reply]
@RCdeWit What value do you try to put in `CRON_MIN` ?

**Reply by RCdeWit** [reply]
Thanks for the quick reply, @Alkarex. Got it to work after a bit of sleep, although I'm still not sure if it's a problem in Portainer or the image.

I deleted the container and created a fresh one, setting `CRON_MIN` to `1,31` on creation. That works fine and I can restart it at will.

Now, when I edit or duplicate the container and update the env var (e.g., `2,32`), I get the same error:

```
OIDC_ENABLED): -c: line 2: unexpected EOF while looking for matching `)'
```

**Reply by Kovah** [reply]
Had the same problem. I noticed that the command in Portainer looked broken. That's where the issue was coming from. I edited the container with the command set to "default". Now it's working again.

**Reply by Frenzie** [reply]
For others reading, broken in which manner?

**Comment by ThoSt81** [comment]
I have the same problem on 2 docker hosts. I'm really lost with this one. A new container starts up fine, I can modify my settings and everything. As soon as I recreate it, I get the issue. I can then only delete the container and set up a new one (pointing at the same volumes). I dont lose any data but it's annoying, especially when the container is restarted for maintenance or anything else.

**Reply by Alkarex** [reply]
@ThoSt81 What Docker image, more precisely? On which platform? And what do you mean by re-create? `docker restart` ?

**Reply by ThoSt81** [reply]
Sorry for the late response. I am using Portiner and I simply "Duplicate/Edit" the container as I do with many of my other containers.
I have now resolved this by starting FreshRSS as a Stack. This appears to work fine.

**Comment by Alkarex** [comment, marked as answer]
Compatibility fix in https://github.com/FreshRSS/FreshRSS/pull/7861
Tests welcome
